extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -305,6 +305,10 @@ export class AmpAnalytics extends AMP.BaseElement {
           // replace selector and selectionMethod
           if (this.isSandbox_) {
             // Only support selection of parent element for analytics in scope
+            if (!this.element.parentElement) {
+              // In case parent element has been removed from DOM, do nothing
```

Comment:
How would this happen?

Reply:
We do call `removeElement`. For example, user remove sticky ad that has analytics component as a children.